### PR TITLE
Add support of GraphQLList of GraphQLLists

### LINKED ISSUE #637: GraphQLList of GraphQLLists?
Good day, I'm having an issue where Android Studio is giving the error `Error:Execution failed for task ':app:generateDebugApolloClasses. com.squareup.javapoet.ParameterizedTypeName cannot be cast to com.squareup.javapoet.ClassName'` when trying to build a query with the following member.
```json
              "name": "myList",
              "description": null,
              "args": [],
              "type": {
                "kind": "LIST",
                "name": null,
                "ofType": {
                  "kind": "LIST",
                  "name": null,
                  "ofType": {
                    "kind": "LIST",
                    "name": null,
                    "ofType": {
                      "kind": "SCALAR",
                      "name": "Float",
                      "ofType": null
                    }
                  }
                }
              }
```

Only happens when there is a LIST of LISTs. Just wondering is this an issue or is there something that needs to be done on my end? I have tried running the query on my local GraphQL and it has no problems fyi. Thanks for any help.

### PR DESCRIPTION
**Pull request by sav007**
Closes #637

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,6 +5,7 @@ subprojects {
     repositories {
       jcenter()
       maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+      mavenLocal()
```
Comment (digitalbuddha) [inline build.gradle]: intentional?

Comment (sav007) [reply build.gradle]: it helps to run integration tests inside IDE with locally installed new version (new changes) of plugin. But yeah will remove it.

**Comment by sav007** [comment]
@digitalbuddha any other comments?

**Comment by digitalbuddha** [comment]
Nope Thank you for this. Web devs will be happy.

**Pull request merged**